app/main.py

```diff
@@ -4,0 +1,65 @@
+from datetime import datetime
+import json
+
+from app.shop import Shop
+from app.customer import Customer
+
+
+def shop_trip() -> None:
+    with open("config.json") as config:
+        config_data = json.load(config)
+    fuel_prize = config_data["FUEL_PRICE"]
+
+    list_customers = [Customer(customers["name"],
+                               customers["product_cart"],
+                               customers["location"],
+                               customers["money"],
+                               customers["car"]
+                               ) for customers in config_data["customers"]]
+
+    list_shops = [Shop(
+        shops["name"],
+        shops["location"],
+        shops["products"])
+        for shops in config_data["shops"]]
+
+    for customer in list_customers:
+        customer.customer_has_money()
+        costs = float("inf")
+        shop_obj = None
+        for shop in list_shops:
+            min_cost = (customer.cost_of_the_trip(
+                fuel_prize,
+                customer.location,
+                shop.location) * 2 + shop.price_products(
+                    customer.product_cart["milk"],
+                    customer.product_cart["bread"],
+                    customer.product_cart["butter"]
+            ))
+            if min_cost < costs:
+                costs = min_cost
+                shop_obj = shop
+            print(f"{customer.name}'s trip to "
+                  f"the {shop.name} costs {min_cost}")
```

Comment:
Consider using double quotes for strings to maintain consistency throughout the code, as recommended by the checklist.